cardano-api/src/Cardano/Api/KeysShelley.hs

```diff
@@ -64,6 +65,8 @@ import           Cardano.Api.SerialiseRaw
 import           Cardano.Api.SerialiseTextEnvelope
 import           Cardano.Api.SerialiseUsing
 
+import           Data.Either.Combinators (maybeToRight)
```

Comment:
Likewise, should be with other `Data.*` modules

Reply:
Fixed.  Thanks!